shielddash/core/templates/shielddash/base.html

```diff
@@ -0,0 +1,17 @@
+{% load staticfiles %}
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="google-signin-client_id" content="263509446425-83g1gv3eus8qe7ss2hgr6ec7hc6krkaf.apps.googleusercontent.com">
```

Comment:
If this will live in Django we should declare this as a constant elsewhere and then inject it here. It shouldn't change often but it would be nice to have a single point of reference.